include/audio_player.h

```diff
@@ -149,10 +151,13 @@ typedef enum {
 } AUDIO_PLAYER_MUTE_SETTING;
 
 typedef esp_err_t (*audio_player_mute_fn)(AUDIO_PLAYER_MUTE_SETTING setting);
+typedef esp_err_t (*audio_reconfig_std_clock)(uint32_t rate, uint32_t bits_cfg, i2s_slot_mode_t ch);
+typedef esp_err_t (*audio_player_write_fn)(char * audio_buffer, uint32_t len, uint32_t *bytes_written, uint32_t timeout_ms);
```

Comment:
should use void* for the audio_buffer type

Reply:
size_t for len and bytes_written

Reply:
yeah, your suggestion is better;